### ISSUE TITLE
How to train your project in multiple GPU?

### ISSUE THREAD

**Issue by John1231983** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#24]
Hello,  I have 4 GPU 1080. Could you tell me how can I train your project in my server because I will use bigger batch size? I have tried uncommend the line to allow growth gpu in your main.py but i think we need to modify the gradient update (average them). I am using tf 1.2.1. Thanks

**Comment by zhengyang-wang** [comment]
I didn't explore any multiple GPU programs so I've no idea. Tensorflow supports model parallelization and data parallelization. Usually model parallelization needs some specific modifications for the code.
